.github/workflows/rust.yml

```diff
@@ -226,6 +240,16 @@ jobs:
         git checkout -f $GITHUB_SHA
         git clean -fd
 
+    - name: Checkout toolchain
```

Comment:
This isn't actually checking out the toolchain, tough, correct? Only the toolchain configuration.